core/common/token-core/build.gradle.kts

```diff
@@ -18,8 +18,10 @@ plugins {
 }
 
 dependencies {
+    api(project(":spi:common:token-spi"))
     api(project(":spi:common:jwt-spi"))
 
+    implementation(project(":extensions:common:crypto:crypto-core")) // for the CryptoConverter
```

Comment:
given that `crypto-core` is not an extension but a library, it could stay in the `core` folder

Reply:
currently `core` contains also library modules (as `util`), anyway I'm ok for leaving it between the extensions as well